### PR TITLE
Feature/add styling

### PR DESCRIPTION
**Pull request by shahadat3669**
### Project Requirement
- [x] Use the CSS rules detailed in Zeplin to style the website.
- [x] Design should match [Zeplin's design](https://pedantic-beaver-cb923b.netlify.app/) (this includes all elements + the same styling)

### General requirements
- [x] There are [no linter errors](https://github.com/microverseinc/linters-config).
- [x] Used correct flow [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- [x] Documented my work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
- [x] Followed list of the [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
- [x] Followed list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

### CODE REVIEW

File: src/components/books/Book.jsx [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
+import 'react-circular-progressbar/dist/styles.css';
 
 import styles from './Book.module.scss';
 import { deleteBook } from '../../redux/books/books';
 
 const Book = ({ book }) => {
-  const {
-    item_id: itemID, author, title, category, completed,
-  } = book;
+  const { item_id: itemID, author, title, category, completed } = book;
```
Comment (zieeco) [inline src/components/books/Book.jsx]: - Please, kindly update these lines of code to the way it was before as this is throwing a linter error and preventing the app to run to fix this issue:

Screenshot from 2023-01-28 13-26-53

Comment (zieeco) [reply src/components/books/Book.jsx]: ### FORM TITLE AND HEADER

- Please, kindly use the correct color for the form title and the header of your app to match the design as much as you can below:
- Use the correct font family as yours is very obvious that it's different compared to the design.


Screenshot from 2023-01-28 13-37-38
Screenshot from 2023-01-28 13-36-42

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @shahadat3669,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by shahadat3669** [comment]
Changes ♻️

- [x] Update Book.js component to fix linters.
- [x] Update Header logo text color.
- [x] Update Add book title text color.

File: src/components/books/Book.module.scss [hunk 2]
```diff
@@ -14,13 +14,32 @@
...
+    font-weight: 700;
+    color: #121212;
+  }
+
+  &__categories {
+    font-weight: 700;
+    color: #121212;
+    font-family: 'Montserrat', sans-serif;
+    font-size: 0.875rem;
+    opacity: 0.5;
```
Comment (Qoosim) [inline src/components/books/Book.module.scss:28]: - [ ] As mentioned by the previous reviewer, It is required to produce the exact design of the sample template. This includes the layout, color, font-family among others. At this moment, your font-family is different from the one used on the sample template and the color of some elements. Kindly look into the two designs and work your design alongside the sample template. All the best.

### Project Requirement.
project-rqts

### Your Design
your-page

### Sample Template
temp-page

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @shahadat3669, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Git Flow. :heavy_check_mark: 




 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/books/Book.module.scss (hunk 2 above)
Comment (shahadat3669) [reply src/components/books/Book.module.scss:28]: @zieeco thanks for your recommendation.

**Comment by shahadat3669** [comment]
### Changes ♻️

- Update font family and colors.

File: src/components/books/Book.module.scss (hunk 2 above)
Comment (Abenezer-Tilahun) [reply src/components/books/Book.module.scss:28]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @shahadat3669 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**